src/main/resources/application.yaml

```diff
@@ -54,7 +54,7 @@ send-letter:
 idam:
   url: ${IDAM_API_URL:http://localhost:5000}
   s2s-auth:
-    totp_secret: ${IDAM_S2S_AUTH_TOTP_SECRET:AAAAAAAAAAAAAAAC}
+    totp_secret: ${IDAM_S2S_AUTH_TOTP_SECRET:}
```

Comment:
This is going to break local development

Reply:
That should be ok to leave there. Let's try...